discovery-infra/test_infra/tools/terraform_utils.py

```diff
@@ -22,6 +23,7 @@ def __init__(self, working_dir: str, terraform_init: bool = True):
         if terraform_init:
             self.init_tf()
 
+    @retry(exceptions=Exception, tries=10, delay=10)
```

Comment:
```suggestion
    @retry(exceptions=python_terraform.TerraformCommandError, tries=10, delay=10)
```
I'm in favor of being more specific at first, so in case we have more basic error (syntax, or just another bug of ours) we will fail immediately

Reply:
done!

Reply:
Thank you! ☺️